### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by menyagah**
# In this section: 
 - I created a collection that keeps a list of books .
 - I created a function to add a new book to the collection, with title and author.
 - I created a function to remove a book from the collection using the array filter() method.
 - I displayed all books saved in the collection in the top part of the page.
 - I made sure that when a user clicks the "Add" button:
    -  A new book is added to the collection.
    -  The new book is displayed on the page.
 - I made sure that when a user clicks the "Remove" button:
    - The correct book is removed from the collection.
    - The correct book disappears from the page.
 - I made sure that data is preserved in the browser's memory using localStorage.
 - I did not use any library or framework, use only plain JavaScript.
 - I fixed all linter errors
 - Below is a link to the live version:
    https://raw.githack.com/menyagah/AwesomeBooks/feature/index.html
 - Made changes on validation as requested by code reviewer

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+addBtn.addEventListener('click', () => {
+  const books = JSON.parse(localStorage.getItem('book') || '[]');
+  const id = Math.random().toString(36).substr(0, 5);
+  books.push({ title: inputTitle.value, author: inputAuthor.value, id });
+  localStorage.setItem('book', JSON.stringify(books));
+  displayBooks();
+  inputAuthor.value = '';
+  inputTitle.value = '';
+});
```
Comment (Emmyn5600) [inline app.js]: - [ ] Everything is working perfectly, however let us ensure that we are not adding empty data in our array, kindly validate the input fields
cpppp1

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @menyagah 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
 - The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
-  Git/GitHub flow best practices have been followed.:heavy_check_mark:
- Add, Remove button as well as localStorage are working properly:heavy_check_mark:
cpppp2

Changes required :writing_hand: 
  
   - Please check the comment under the review.

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: app.js (hunk 1 above)
Comment (menyagah) [reply app.js]: Thank you so much for the review. I am working on it ASAP.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by menyagah** [comment]
Thank you @vichuge !

**Pull request merged**